meerk40t/core/elements/shapes.py

```diff
@@ -740,36 +740,47 @@ def recalc(command, channel, _, data=None, post=None, **kwargs):
         self.signal("refresh_scene", "Scene")
         self.validate_selected_area()
 
+    @self.console_option(
+        "tolerance", "t", type=float, help=_("simplification tolerance"),
```

Comment:
Could just add a `default=25` here rather than getting None and using that to set it. It's supposed to be required to pass that to the function if the default it set.